app/assets/javascripts/controllers/ops/tenant_form_controller.js

```diff
@@ -13,21 +14,24 @@ ManageIQ.angularApplication.controller('tenantFormController', ['$http', '$scope
       ManageIQ.angularApplication.$scope = $scope;
 
       if (tenantFormId == 'new') {
-        $scope.newRecord               = true;
-        $scope.tenantModel.name        = '';
-        $scope.tenantModel.description = '';
-        $scope.tenantModel.default     = false;
-        $scope.tenantModel.divisible   = tenantType;
-        $scope.afterGet                = true;
-        $scope.modelCopy               = angular.copy( $scope.tenantModel );
+        $scope.newRecord                             = true;
+        $scope.tenantModel.name                      = '';
+        $scope.tenantModel.description               = '';
+        $scope.tenantModel.default                   = false;
+        $scope.tenantModel.divisible                 = tenantType;
+        $scope.tenantModel.use_config_for_attributes = null;
```

Comment:
for some reason when defaulting this to false, was causing Save/reset buttons to light up when going to edit of other tenant records, i think it will be good if you can default it to false in the model.